tests/system-test/fulltest.sh

```diff
@@ -147,7 +147,7 @@ python3 ./test.py -f 2-query/query_cols_tags_and_or.py
 
 python3 ./test.py -f 2-query/elapsed.py
 python3 ./test.py -f 2-query/csum.py
-python3 ./test.py -f 2-query/mavg.py
```

Comment:
@wenzhouwww mavg.py commented out. Please add test cases to cover mavg with selective column cases after this PR got merged.